### DISCUSSION TITLE
打开gb2312编码的中文会显示乱码

### DISCUSSION THREAD

**Discussion by colorthoro** [cpeditor/cpeditor-cn#2] [提问] [answered]
如题，怎么办？

**Comment by ouuan** [comment, marked as answer]
我平常都用 Linux，所以不太熟悉 Windows 的软件生态，也不会碰到 gb2312 的文件...我暂且猜测一下解决办法。

1. 你可以试一试用支持 gb2312 的编辑器打开，然后复制到 CP Editor 里。
2. 你可以用有格式转换功能的文本编辑器（好像记事本就可以，另存为的时候选择编码）转换格式。

但是，更好的做法是不要用中文写注释，或者只用 CP Editor 以及其它使用 UTF-8 编码的 IDE。

在主仓库有支持多种编码的 issue: https://github.com/cpeditor/cpeditor/issues/319 和一个有一些问题的 PR: https://github.com/cpeditor/cpeditor/pull/321
